src/App.tsx

```diff
@@ -1,13 +1,25 @@
-
-
-
+import { Link } from "react-router-dom";
+import axios from "axios";
+import { useEffect, useState } from "react";
 export default function App() {
   
-
+  const [randomURL, setRandomURL] = useState("")
+   useEffect(() => {
+      axios.get("http://localhost:4000/randomid")
+     .then((res) => {
+      setRandomURL(res.data.string)
+     }).catch(e => {
+       console.log(e)
```

Comment:
```suggestion
       console.error('getSlug:', e)
```

We can use console.error instead of log. We should also name the error to identify where the error comes from